File: C++/no4.cpp
#include <iostream>

using namespace std;

int main() {
  int td;

  cout << "Masukkan total detik: ";
  cin >> td;

  int jam = td / 3600;
  int menit = (td % 3600) / 60; 
  int detik = td % 60;

  cout << "Jam: " << jam << endl;
  cout << "Menit: " << menit << endl;
  cout << "Detik: " << detik << endl;

  return 0;
}

File: PHP/no4.php
<?php
  $td = 5440;

  $jam = $td / 3600;
  $menit = ($td % 3600) / 60; 
  $detik = $td % 60;

  echo"Jam: " .$jam;
  echo"Menit: " .$menit;
  echo"Detik: " .$detik;

  return 0;
?>